tests/snuba/api/endpoints/test_organization_events_stats.py

```diff
@@ -725,6 +725,31 @@ def setUp(self):
             kwargs={"organization_slug": self.project.organization.slug},
         )
 
+    def test_no_top_events(self):
+        project = self.create_project()
+        with self.feature(self.enabled_features):
+            response = self.client.get(
+                self.url,
+                data={
+                    # make sure to query the project with 0 events
+                    "project": project.id,
+                    "start": iso_format(self.day_ago),
+                    "end": iso_format(self.day_ago + timedelta(hours=2)),
+                    "interval": "1h",
+                    "yAxis": "count()",
+                    "orderby": ["-count()"],
+                    "field": ["count()", "message", "user.email"],
+                    "topEvents": 5,
+                },
+                format="json",
+            )
+
+        data = response.data["data"]
+        assert response.status_code == 200, response.content
+        # When there are no top events, we do not return an empty dict.
+        # Instead, we return a single zero-filled series for an empty graph.
+        assert [attrs for time, attrs in data] == [[{"count": 0}], [{"count": 0}]]
```

Comment:
Should the response shape stay the same as if a topN hit was found? You could use '' as the series key.

Reply:
Using the empty string as the series key will keep the response shape the same but there are some drawbacks.

The empty string is technically a valid value, and thus a potential series name, so we won't have any way of distinguishing between the two. This means that we can't label the tooltip properly for a series with the empty string as the name vs a no results series.